src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE_OF_ARRAY = 10;
+    private K[] keyArr;
+    private V[] valueArr;
+    private int sizeOfArr;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
    private int size;
```

dont use shortenings